internal/service/neptune/sweep.go

```diff
@@ -33,6 +34,9 @@ func init() {
 	resource.AddTestSweepers("aws_neptune_cluster_instance", &resource.Sweeper{
 		Name: "aws_neptune_cluster_instance",
 		F:    sweepClusterInstances,
+		Dependencies: []string{
+			"aws_rds_global_cluster",
```

Comment:
Should this be `aws_neptune_global_cluster`?

Reply:
Surprisingly, no. It looks like a Neptune global cluster is implemented using an RDS Global Cluster. I'll add a comment to clarify that.